Cargo.toml

```diff
@@ -13,12 +13,13 @@ edition = "2018"
 default = ["std", "multihash/default"]
 std = ["multihash/std", "unsigned-varint/std", "alloc", "multibase/std"]
 alloc = ["multibase", "multihash/alloc", "core2/alloc", "serde/alloc"]
-arb = ["quickcheck", "rand", "multihash/arb"]
+arb = ["quickcheck", "rand", "multihash/arb", "arbitrary"]
 scale-codec = ["parity-scale-codec", "multihash/scale-codec"]
 serde-codec = ["alloc", "serde", "multihash/serde-codec", "serde_bytes"]
 
 [dependencies]
-multihash = { version = "0.16.1", default-features = false }
+#multihash = { version = "0.16.1", default-features = false }
+multihash = { git = "https://github.com/multiformats/rust-multihash", branch = "master" }
```

Comment:
The errors on CI come from the missing `default-features = false`.

Reply:
Aaah, no idea then why it also fails on master locally but it should fix it.